.github/workflows/sanity-check.yml

```diff
@@ -0,0 +1,29 @@
+# Workflow that validates a AOD requests.
+name: 'aod-validate'
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'iam.yaml'
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-aod-request-validation'
+  cancel-in-progress: true
+
+jobs:
+  # An always fail job to prevent AOD request from merging to main.
+  do_not_merge:
+    runs-on: 'ubuntu-latest'
+    steps:
```

Comment:
Since we already have a path constraint on the trigger, don't really need to check file existence here.